LoopKit/InsulinKit/InsulinDeliveryStore.swift

```diff
@@ -454,3 +454,26 @@ extension InsulinDeliveryStore {
         }
     }
 }
+
+extension NSManagedObjectContext {
+    fileprivate func cachedInsulinDeliveryObjectsWithUUID(_ uuid: UUID, fetchLimit: Int? = nil) -> [CachedInsulinDeliveryObject] {
```

Comment:
see [nit] comment above about prefixing this with `fetchCached...`

Reply:
See above response to above comment. ;)